kits/sql-query-generator/apps/.env.example

```
@@ -0,0 +1,4 @@
+SQL_QUERY_GENERATOR_FLOW_ID = "YOUR_FLOW_ID"
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)